### PR TITLE
fix: adding the uri to the env

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes

Fixes the environment return on generateDID for Attester as it is required for the verification but not logged out.

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: code_examples/sdk_examples/src/workshop/attester/generateDid.ts [hunk 1]
```diff
@@ -61,10 +61,11 @@ if (require.main === module) {
       // Load attester account
       const accountMnemonic = process.env.ATTESTER_ACCOUNT_MNEMONIC as string
       const { account } = generateAccount(accountMnemonic)
-      const { mnemonic } = await createFullDid(account)
+      const { mnemonic, fullDid } = await createFullDid(account)
 
       console.log('\nsave following to .env to continue\n')
       console.error(`ATTESTER_DID_MNEMONIC="${mnemonic}"\n`)
+      console.error(`ATTESTER_DID_URI="${fullDid.uri}"\n`)
```
Comment (rflechtner) [inline code_examples/sdk_examples/src/workshop/attester/generateDid.ts:68]: this should be mentioned in the text (i.e. in the markdown) as well, along with the mnemonic

Comment (Dudleyneedham) [reply code_examples/sdk_examples/src/workshop/attester/generateDid.ts:68]: Agreed. I have updated

Comment (rflechtner) [reply code_examples/sdk_examples/src/workshop/attester/generateDid.ts:68]: I was talking specifically about line 93-95 in the file below, that should be updated to reflect that we've got two env variables being logged to console now

Comment (Dudleyneedham) [reply code_examples/sdk_examples/src/workshop/attester/generateDid.ts:68]: Understood :)

**Review by rflechtner — APPROVED** [review]
looks good to me now!

**Pull request merged**